framework_tvb/tvb/interfaces/web/controllers/users_controller.py

```diff
@@ -128,12 +128,12 @@ def profile(self, logout=False, save=False, **data):
                 form = EditUserForm()
                 data = form.to_python(data)
                 if data.get(KEY_PASSWORD):
-                    user.password = md5(data[KEY_PASSWORD]).hexdigest()
+                    user.password = md5(data[KEY_PASSWORD].encode('utf-8')).hexdigest()
```

Comment:
Have you tested from web GUI with start clean ?

Reply:
Yes. I was able to create a new user, log in, change password then log in again in a clean environment and checked by debugging if those values are obtained correctly with the encoding.